reports/get_stats.rmd

```diff
@@ -0,0 +1,310 @@
+---
+title: "Youthrex data table script"
```

Comment:
Do you use anything from this script? I know i wrote some of this code, but I can't remember which one is mine.